pkg/interface/src/views/apps/chat/ChatResource.tsx

```diff
@@ -134,6 +134,11 @@ function ChatResource(props: ChatResourceProps) {
     api.graph.addPost(ship, name, createPost(window.ship, contents))
   }, [resource]);
 
+  const onDelete = useCallback((msg: Post) => {
+    const { ship, name } = resourceFromPath(resource);
+    api.graph.removePosts(ship, name, [msg.index]);
+  }, [resource]);
```

Comment:
missing `msg.index` as a dependency

Reply:
Why is msg.index a dependency? We pass msg into the callback.

Reply:
nvm!